### PR TITLE
Rotisserie chick sweet potato

### PR DESCRIPTION
**Pull request by diego-codes**

### CODE REVIEW

**Review by NicMoe — CHANGES_REQUESTED** [review]
Rotisserie chicken and sweet potatoes are two different things. Maybe we could consider a section called "meals" where we recommend multiple recipes together, but I think we should split these up.

**Comment by diego-codes** [comment]
Thanks,  @NicMoe this is ready to review again.

File: meals/rotisserie-chicken-sweet-potato.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Rotisserie chicken with sweet potatoes
+
+This recipe is inspired by the delicious food from [Fresas in Austin, TX](https://www.fresaschicken.com/). 
+
+
+## Ingredients for two people
+
+- 1 store-bought rotisserie chicken
+- Roasted sweet potato havles ([recipe](../sides/roasted-sweet-potato-halves.md))
```
Comment (NicMoe) [inline meals/rotisserie-chicken-sweet-potato.md]:
```suggestion
- Roasted sweet potato halves ([recipe](../sides/roasted-sweet-potato-halves.md))
```

File: meals/rotisserie-chicken-sweet-potato.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Instructions
+
+1. Follow directions to roast the veggies.
+
+2. While the veggies are cooking, cut the rotisserie chicken into pieces with a sharp large knife.
+
+3. Once the veggies are done cooking, heat the tortillas on a hot skillet according to their instructions.
+
+7. Serve veggies, chicken, tortillas, and sauce. Enjoy!
```
Comment (NicMoe) [inline meals/rotisserie-chicken-sweet-potato.md]:
```suggestion
4. Serve veggies, chicken, tortillas, and sauce. Enjoy!
```

File: sides/roasted-sweet-potato-halves.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Roasted sweet potato halves
+
+## Ingredients for two people
+
+- 2 large sweet potatoes (or 4-6 small sweet potatoes) cut in half
+- 1 medium white or yellow onion cut in quarters
+- 6 garlic cloves whole and peeled
+- Olive oil
```
Comment (NicMoe) [inline sides/roasted-sweet-potato-halves.md]: How much oil, salt, pepper, and spices?

File: sides/roasted-sweet-potato-halves.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Aluminum foil
+- Baking tray
+
+
+## Instructions
+
+1. Preheat the oven to 425°F.
+
+2. Coat all prepped veggies (sweet potato havles, onion quarters, and whole garlic cloves) with olive oil and salt, pepper, and spices to taste.
```
Comment (NicMoe) [inline sides/roasted-sweet-potato-halves.md]:
```suggestion
2. Coat all prepped veggies (sweet potato halves, onion quarters, and whole garlic cloves) with olive oil and salt, pepper, and spices to taste.
```

Comment (NicMoe) [reply sides/roasted-sweet-potato-halves.md]: How much of each of these is recommended? "To taste" is nice, but giving readers a baseline to modify from is better.

**Comment by diego-codes** [comment]
This PR is ready to review again, @NicMoe. Thank you!

**Review by NicMoe — APPROVED** [review]

**Pull request merged**